tests/unit_tests/test_helm_chart.py

```diff
@@ -305,3 +293,71 @@ def group_manifests(ungrouped: Iterable[Mapping[str, Any]]) -> GroupedManifests:
             )
         group[name] = manifest
     return groups
+
+
+def test_init_container_config_not_available_when_disabled():
```

Comment:
Nit: The new tests should go above the utility functions like `render_chart` et al.